src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -146,6 +169,8 @@
     private int topCitizenId = 0;
     private int maxCitizens  = Configurations.maxCitizens;
 
+    private double overallHappiness = 5;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)